code/clue.js

```diff
@@ -1,59 +1,342 @@
+  name: 'Study room',
+  image: 'https://i5.walmartimages.com/asr/903edd24-c5f6-4b03-b337-496fe699db2b.a36e87cc654a3a629f75fa39a32e09d2.png'
+};
+
+const libraryRoom = {
+  name: 'Library Room',
+  image: 'https://i.pinimg.com/474x/6f/4f/55/6f4f5570f5abf34ddbbd23f09878041a.jpg'
+};
+
+const billiardRoom = {
+  name: 'Billiard Room',
+  image: 'https://imgproxy.geocaching.com/a308ed3322b4a879e42c8a79c46599c38decb956?url=http%3A%2F%2Ffarm4.static.flickr.com%2F3025%2F2684464802_4f2276f1c6_o.jpg'
+};
+
+const lounge = {
+  name: 'Lounge',
+  image: 'https://i.pinimg.com/474x/63/03/19/630319400f1fcc7032e1667ee3672491.jpg'
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const ballRoom = {
+  name: 'Ballroom',
+  image: 'https://i.pinimg.com/originals/18/cc/05/18cc059f3aa976ddd543222a533a2f5f.jpg'
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const hall = {
+  name: 'Hall',
+  image: 'https://st.hzcdn.com/simgs/6961a9ba0d4331b3_4-4724/home-design.jpg'
+};
 
+const spa = {
+  name: 'Spa',
+  image: 'https://www.teamtoyboxes.com.au/wp-content/uploads/2019/05/Spa-card.jpg'
+};
+
+const livingRoom = {
+  name: 'Living Room',
+  image: 'https://cdn.concreteplayground.com/content/uploads/2019/03/Baedeker_FB-1920x1440.jpg'
+};
+
+const observatory = {
+  name: 'Observatory',
+  image: 'https://i.pinimg.com/originals/87/ef/6c/87ef6c8d51be485eb9a5e2129de04fb4.jpg'
+};
+
+const theater = {
+  name: 'Theater',
+  image: 'https://i.pinimg.com/originals/0e/f6/e6/0ef6e620320017d4c4f9b4b0e931dc77.jpg'
+};
+
+const guestHouse = {
+  name: 'Guest House',
+  image: 'https://www.thesun.co.uk/wp-content/uploads/2019/12/60a659b1-5983-43ae-a6c0-5a2624d0384bjpg-JS550206142.jpg'
+};
+
+const patio = {
+  name: 'Patio',
+  image: 'https://steveandamysly.tannerworld.com/databank/2009/image_clue_reinvention04.jpg'
+};
+
+// GROUPING OF ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS
 const suspects = [
   mrGreen,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
   mrsWhite
-  // ...  and the rest
-]
+];
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+];
 
-const rooms = []
+const rooms = [ 
+  diningRoom, 
+  conservatory, 
+  kitchen, 
+  studyRoom, 
+  libraryRoom,
+  billiardRoom, 
+  lounge, 
+  ballRoom,
+  hall,
+  spa,
+  livingRoom,
+  observatory, 
+  theater, 
+  guestHouse, 
+  patio
+];
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
+  return array[Math.floor(Math.random() * array.length)];
+};
+
+// CREATE AN OBJECT THAT KEEPS THE MYSTERY. With a killer, a weapon and a room. The values are set later down.
+let theMystery = { // Write mystery instead?
+ killer: '',
+ weapon: '',
+ room: ''
```

Comment:
Prefer these to start as `null` values.  This the usual Javascript information for "no value"